src/server/CMakeLists.txt

```diff
@@ -61,6 +61,10 @@ if (PRINT_STACKTRACES_ON_SIGNAL)
   target_compile_definitions(dragonfly_lib PRIVATE PRINT_STACKTRACES_ON_SIGNAL)
 endif()
 
+if (WITH_ASAN OR WITH_USAN)
+  target_compile_definitions(dfly_transaction PRIVATE SANITIZERS)
+endif()
+
```

Comment:
Aren't there flags to decect it witout exporing another define? Like `__SANITIZE_ADDRESS__` that absl uses (and even we do)

Reply:
Yes there is but it's ugly because clang and gcc use different macros. So it would look like this:

```
#if defined(__clang__)
#if defined(__has_feature)
#if __has_feature(address_sanitizer)

#else
#ifdef __SANITIZE_ADDRESS__
....
```

Which it's monstrous :rofl: In fact, I should refactor the above tbh at some point